frontend/web/styles/components/_panel.scss

```
@@ -121,8 +121,12 @@
         }
       }
     }
-    .list-item:hover a,.btn-link {
-      color: $primary !important;
+    // :hover is spelled out because `a:hover` above is also !important and
```

Stylelint (17.14.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)